src/Unleash/Internal/UnleashServices.cs

```diff
@@ -35,14 +36,16 @@ internal class UnleashServices : IDisposable
             "userWithId"
         };
 
-        public UnleashServices(UnleashSettings settings, EventCallbackConfig eventConfig, List<IStrategy> strategies = null)
+        public UnleashServices(UnleashSettings settings, EventCallbackConfig eventConfig, List<Strategies.IStrategy> strategies = null)
         {
             if (settings.FileSystem == null)
             {
                 settings.FileSystem = new FileSystem(settings.Encoding);
             }
 
-            engine = new YggdrasilEngine(strategies);
+            List<Yggdrasil.IStrategy> yggdrasilStrategies = strategies?.Select(s => new CustomStrategyAdapter(s)).Cast<Yggdrasil.IStrategy>().ToList();
```

Comment:
the `s as IStrategy` didn't work here?

Reply:
Ah, never mind. 

> The barebones interface doesn't work because the method signature requires an Yggdrasil type - the Context.

Reply:
I'm not sure what you mean? Does that add anything over the cast?